### PR TITLE
Feature/avo 3165 skip terms of service for elementary pupils

### PR DESCRIPTION
**Pull request by ian-emsens-shd**
- feat(AVO-3165): Don't show terms of service to elementary pupils + scaffold landingpage

### CODE REVIEW

File: src/shared/helpers/user-is.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { CommonUserSchema } from '@viaa/avo2-types/types/user';
+
+import { SpecialUserGroup } from '../../admin/user-groups/user-group.const';
+
+export function userIsPupil(user: CommonUserSchema) {
+	return userIs(user, SpecialUserGroup.Pupil);
```
Comment (bertyhell) [inline src/shared/helpers/user-is.ts:6]: Will this one still be used? Shouldn't all pupils either be ElementaryPupil or SecondaryPupil after the migration?

**Review by bertyhell — APPROVED** [review]

File: src/authentication/views/registration-flow/accept-elementary-pupil-conditions.tsx [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+			</Container>
+			<Spacer margin="large">
+				<Toolbar>
+					<ToolbarCenter>
+						{loading ? (
+							<Spinner size={'large'} />
+						) : (
+							<Button
+								label={tText('TODO')}
+								title={tText('TODO')}
```
[the remark this answers is not in the document]
Comment (ian-emsens-shd) [reply src/authentication/views/registration-flow/accept-elementary-pupil-conditions.tsx:80]: Out of scope for this ticket, https://github.com/viaacode/avo2-client/pull/1981

File: src/shared/helpers/user-is.ts (hunk 1 above)
Comment (ian-emsens-shd) [reply src/shared/helpers/user-is.ts]: User group was renamed, not created again. `SpecialUserGroup.Pupil` will refer to secondary pupils post-migration.

Comment (bertyhell) [reply src/shared/helpers/user-is.ts]: ok, so maybe we should also rename the name of this entry in the enum, but keep the value the same?
```SpecialUserGroup.PupilSecundary```?

Comment (bertyhell) [reply src/shared/helpers/user-is.ts]: Ok, then we should probably rename the enum to PupilSecundary but leave its underlaying value the same

**Pull request merged**